torch/utils/mobile_optimizer.py

```diff
@@ -5,15 +5,18 @@
 import torch
 from enum import Enum
 from torch._C import MobileOptimizerType
-from typing import Set
+from typing import Set, List, AnyStr
 
 class LintCode(Enum):
     BUNDLED_INPUT = 1
     REQUIRES_GRAD = 2
     DROPOUT = 3
     BATCHNORM = 4
 
-def optimize_for_mobile(script_module, optimization_blacklist: Set[MobileOptimizerType] = None):
+def optimize_for_mobile(
+        script_module,
+        optimization_blacklist: Set[MobileOptimizerType] = None,
+        preserved_methods: List[AnyStr] = []):
```

Comment:
I wonder if we should make these keyword-only args.  Seems error-prone to use them as positional args.

Reply:
like this?
`def optimize_for_mobile(scripted_module, **kwargs)`?